fs/fs.go

```diff
@@ -62,6 +63,7 @@ type Context struct {
 	// docker root directory.
 	DockerRoot string
 	DockerInfo map[string]string
+	RktPath    string
```

Comment:
Ok. I started my review at too fine a grain (sorry). We can ignore this fore now.